compat/win32/flush.c

```diff
@@ -8,7 +8,7 @@ int win32_fsync_no_flush(int fd)
 
 #define FLUSH_FLAGS_FILE_DATA_ONLY 1
 
-       DECLARE_PROC_ADDR(ntdll.dll, NTSTATUS, NtFlushBuffersFileEx,
+       DECLARE_PROC_ADDR(ntdll.dll, NTSTATUS, NTAPI, NtFlushBuffersFileEx,
```

Comment:
I'm guessing our `NTAPI` definition here comes from `winternl.h` here. I used `winnt.h` in `t/helper/test-drop-caches.c` we might want to make them use the same source at some point in the future, but this isn't urgent.

Reply:
> I'm guessing our `NTAPI` definition here comes from `winternl.h` here. I used `winnt.h` in `t/helper/test-drop-caches.c` we might want to make them use the same source at some point in the future, but this isn't urgent.

`winternl.h` [`#include`s `windef.h`](https://github.com/git-for-windows/git-sdk-64/blob/4e3578c6d5cf9fc814f3724e500b4c5e78a446fb/mingw64/x86_64-w64-mingw32/include/winternl.h#L9), which in turn [`#include`s `winnt.h`](https://github.com/git-for-windows/git-sdk-64/blob/4e3578c6d5cf9fc814f3724e500b4c5e78a446fb/mingw64/x86_64-w64-mingw32/include/windef.h#L19-L22).

So there is nothing we need to do here.

Besides, we can allow ourselves to trust the `mingw-w64-headers` to _not_ `#define` parts as central as `NTAPI` in incompatible ways depending which header file you `#include`.